packages/node-app/src/tests/auth.controller.test.ts

```diff
@@ -0,0 +1,204 @@
+import fs from 'fs';
+import mongoose from 'mongoose';
+import request from 'supertest';
+
+import { app } from '../app';
+import { giveawaysContract } from '../contracts';
+import { Giveaway } from '../models/giveaway.model';
+import { Location } from '../models/location.model';
+import { Unit, User, UserRole } from '../models/user.model';
+import { getUserInfo } from '../utils/auth.util';
+
+const userInfo = {
+  email: 'example@domain.com',
+  name: 'name',
+  picture: 'picture',
+  unit: Unit.NODE,
+};
+
+giveawaysContract.methods.createGiveaway = jest
+  .fn()
+  .mockReturnValue({ send: () => ({}) });
+
+giveawaysContract.methods.addParticipant = jest
+  .fn()
+  .mockReturnValue({ send: () => ({}) });
+
+jest.mock('../utils/auth.util.ts', () => ({
+  getUserInfo: jest.fn(),
+}));
```

Comment:
I believe we need to mock the module since it does not contain any default import, it just export a few utils functions. I believe it is not possible to mock directly a function that was imported. `Cannot assign to 'getUserInfo' because it is an import`

Reply:
How about `const mockedGetUserInfo = jest.mocked(getUserInfo, true)`?
Here is the documentation with an example: https://jestjs.io/docs/mock-function-api/#jestmockedsource-options